pom.xml

```
@@ -18,7 +18,7 @@
 		<kotlinx-coroutines.version>1.6.2</kotlinx-coroutines.version>
 		<spring-cloud-azure.version>4.0.0</spring-cloud-azure.version>
 		<jacoco.version>0.8.8</jacoco.version>
-		<pagopa-ecommerce-commons.version>0.0.3</pagopa-ecommerce-commons.version>
+		<pagopa-ecommerce-commons.version>0.1.1</pagopa-ecommerce-commons.version>
```

```suggestion
		<pagopa-ecommerce-commons.version>0.2.0</pagopa-ecommerce-commons.version>
```
Use the last commons library revision